superset-frontend/src/views/CRUD/alert/AlertList.tsx

```diff
@@ -183,10 +183,6 @@ function AlertList({
     }
   };
 
-  useEffect(() => {
-    refreshData();
-  }, [isReportEnabled]);
-
```

Comment:
This was causing duplicate api calls when switching back and forth between alerts/reports. We no longer need this as baseFilters is changing which results in a new fetchData, which triggers a new api call.

Reply:
nice catch